torch/autograd/gradcheck.py

```diff
@@ -141,7 +141,70 @@ def get_numerical_jacobian(fn, inputs, outputs=None, target=None, eps=1e-3,
         target = inputs
     inp_indices = [i for i, a in enumerate(target) if is_tensor_like(a) and a.requires_grad]
     for i, (inp, inp_idx) in enumerate(zip(iter_tensors(target, True), inp_indices)):
-        jacobians += [get_numerical_jacobian_helper(fn, inp, inp_idx, inputs, outputs, eps, grad_out)]
+        jacobians += [get_numerical_jacobian_wrt_specific_input(fn, inp, inp_idx, inputs, outputs, eps, eps, grad_out)]
+    return jacobians
+
+
+def compute_numerical_gradient(fn, entry, v, norm_v, nbhd_checks_fn):
```

Comment:
Would it be simpler if we computed `norm_v` inside this function instead of passing it in as an argument? Depending on what `v` can be this is not too computationally expensive